### ISSUE TITLE
Getting error when trying to find cluster/log-dump/log-dump.sh

### ISSUE THREAD

**Issue by gamename** [heptio/sonobuoy#80]
Noticed this error in the e2e log.  This happens with failed or successful tests:
```
[snip]
03: INFO: Running AfterSuite actions on all node
Sep 25 19:38:00.103: INFO: Running AfterSuite actions on node 1
>>>Sep 25 19:38:00.104: INFO: Error running cluster/log-dump/log-dump.sh: fork/exec ../../cluster/log-dump/log-dump.sh: no such file or directory

Ran 1 of 652 Specs in 39.740 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 651 Skipped PASS
```

It doesn't seem to affect the tests, but it does seem like a problem

**Comment by timothysc** [comment]
^ This is known, and not really an issue. 

There are parts of the e2e's that depend on shelling out to scripts in the repository to do some of the work that sonobuoy already does.  e.g. pod-log collection.  

This is something we should aim to fix upstream in the 1.9 series.

**Comment by timothysc** [comment]
Closing, as it's not an issue.
